src/ImageProcessorCore/IO/EndianBinaryReader.cs

```diff
@@ -122,7 +122,7 @@ public void Close()
         /// </summary>
         /// <param name="offset">Offset to seek to.</param>
         /// <param name="origin">Origin of seek operation.</param>
-        public void Seek(int offset, SeekOrigin origin)
+        public virtual void Seek(int offset, SeekOrigin origin)
```

Comment:
LOL...

I guess I missed the forest through the trees on this one. I'm not sure we really need the TiffReader any longer. It's not really doing anything.